app/src/test/java/nl/tudelft/b_b_w/model/BlockFactoryUnitTest.java

```diff
@@ -33,7 +34,7 @@
      */
     @Before
     public void makeNewBlock() throws Exception {
-        _block = new Block(owner, sequenceNumber, ownHash, previousHashChain, previousHashSender, publicKey, isRevoked);
+        _block = new Block(owner, sequenceNumber, ownHash, previousHashChain, previousHashSender, publicKey,iban, isRevoked);
```

Comment:
I did not write BlockFactoryUnitTest. I cannot just change the whole method (which will result in changing other methods and me working on other issue). Someone whos responsible for this class (@ashay123 toch?) should be making an issue and then fix this.